### PR TITLE
Aggregates rates at end of run

### LINKED ISSUE #10: Rundoc update with datarate
We are very often speaking about a certain datarate for a run. Let's add that to the rundoc such that it's easily afterwards to have that number. Now, we need to do some aggregate on the status which is deleted after a week.

Should be only one or two lines in the dispatcher.

### PR DESCRIPTION
**Pull request by darrylmasson**
Fixes #10 .

### CODE REVIEW

**Review by jorana — APPROVED** [review]
Many many thanks @darrylmasson , every time I started writing a suggestion, I looked a bit harder and saw you already did that.

File: dispatcher/MongoConnect.py [hunk 1]
```diff
@@ -362,22 +362,33 @@ def GetNextRunNumber(self):
...
-    def SetStopTime(self, number, detector, force):
+    def SetStopTime(self, number, detectors, force):
         '''
         Sets the 'end' field of the run doc to the time when the STOP command was ack'd
         '''
-        self.log.info("Updating run %i with end time"%number)
+        self.log.info("Updating run %i with end time (%s)" %(number, detectors))
         try:
             time.sleep(2) # this number depends on the delay between CC and reader stop
             endtime = self.GetAckTime(detector, 'stop')
```
Comment (jorana) [inline dispatcher/MongoConnect.py:372]: this should say detectors

Comment (darrylmasson) [reply dispatcher/MongoConnect.py:372]: It probably should, but how many people ever read the dispatcher logs?

Comment (jorana) [reply dispatcher/MongoConnect.py:372]: No I mean `detector` is not defined in `endtime = self.GetAckTime(detector, 'stop')`

Comment (darrylmasson) [reply dispatcher/MongoConnect.py:372]: Ah, you mean a typo. I was looking at the highlighted change lines :rofl:

**Comment by darrylmasson** [comment]
rate_aggregate

**Pull request merged**